mitodl/mitxonline
# Showing Course rather than Courses (or program/s) when there is only 1 to display

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/2488

### Description (What does it do?)
Adds a function to render the correct display text on the desktop view for 1 course or program as opposed to multiple (without an s to make the noun plural).

### How can this be tested?
Create a department that only has one course and/or one program assigned. You should see the text change from 1 courses (in current main branch) to 1 course when you select that department on the catalog page.
